source/recipe-individual/scripts/individualCustom.js

```diff
@@ -0,0 +1,301 @@
+            const single = document.createElement("li");
+            single.innerHTML = data.extendedIngredients[i].original;
+            ingredientsText.appendChild(single);
+        }
+        rightContainer.appendChild(thumbnail);
+        ingredientsContainer.appendChild(ingredientsHeadline);
+        ingredientsContainer.appendChild(ingredientsText);
+        rightContainer.appendChild(ingredientsContainer);
+
+        const topMiddleContainer = document.createElement('div');
+        topMiddleContainer.setAttribute("class", 'image-box');
+        const recipeTitle = document.createElement('h1');
+        recipeTitle.innerHTML = data.title;
+        const cookTime = document.createElement('h2');
+        cookTime.innerHTML = data.readyInMinutes + "min";
+        // const description = document.createElement('p');
+        // description.innerHTML = data.
+        const ratingBox = document.createElement("div");
+        ratingBox.setAttribute("class", 'ratings-box');
+        const starsDiv = document.createElement('div');
+        const stars = document.createElement("img");
+        stars.setAttribute('src', './images/5-star.svg');
+        stars.setAttribute('class', 'stars');
+        starsDiv.appendChild(stars);
+        const ratingsDiv = document.createElement("div");
+        const ratings = document.createElement('p');
+        ratings.innerHTML = data.aggregateLikes + " Likes";
+        ratingsDiv.appendChild(ratings);
+        const favorite = document.createElement('button');
+        favorite.setAttribute('class', 'favorite');
+        favorite.setAttribute('id', 'favorite');
+        favorite.addEventListener("click", function() {
+            var thumbnail = {"totalTime" : data.readyInMinutes, 
+            "title" : data.title, "id": data.id, "thumbnailUrl": data.image}
+            localStorage.setItem(data.title, JSON.stringify(thumbnail))
+            console.log(localStorage);
+            });
+        const favImage = document.createElement("img");
+        favImage.setAttribute('src', 'images/favorite.png');
+        favImage.setAttribute('height', '70vh');
+        favImage.setAttribute('widht', '100vw');
+        favorite.appendChild(favImage);
+        const favoriteDiv = document.createElement("div");
+        favoriteDiv.appendChild(favorite);
+        topMiddleContainer.appendChild(recipeTitle);
+        topMiddleContainer.appendChild(cookTime);
+        ratingBox.appendChild(starsDiv);
+        ratingBox.appendChild(ratingsDiv);
+        ratingBox.appendChild(favoriteDiv);
+        topMiddleContainer.appendChild(ratingBox);
+
+        const bottomMidContainer = document.createElement('div');
+        bottomMidContainer.setAttribute('class', 'directions-text');
+        const instructionsHeader = document.createElement('h2');
+        instructionsHeader.innerHTML = 'Instructions';
+        const instructions = document.createElement("ul");
+        const betterInstructions = data.instructions.split('<p>');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445170)
```suggestion
        const betterInstructions = data.instructions.split("<p>");
```